android/src/main/java/com/clevertap/clevertap_plugin/CleverTapPlugin.java

```diff
@@ -822,6 +841,28 @@ private Map<String, Object> getVariableValueAsMap(String name) {
                 "Variable name = " + name + " does not exist.");
     }
 
+    private void getAppLaunchNotification(Result result) {
```

Comment:
@shivamsharma2710 add comments on what this code does

Reply:
Resolved!